react/src/components/UserAvatar/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import strings from './strings';
+import './styles.css';
+import default_avatar from '../../assets/default-avatar.gif';
+
+class UserAvatar extends Component {
```

Comment:
I'd write this as a stateless component (i.e. as a function), since it only renders an element.